src/solvers/dg/1d/dg.jl

```diff
@@ -0,0 +1,1411 @@
+# Main DG data structure that contains all relevant data for the DG solver
+mutable struct Dg1D{Eqn<:AbstractEquation, NVARS, POLYDEG,
+                  SurfaceFlux, VolumeFlux, InitialConditions, SourceTerms, BoundaryConditions,
+                  VolumeIntegralType, ShockIndicatorVariable,
+                  VectorNnodes, MatrixNnodes, MatrixNnodes2,
+                  InverseVandermondeLegendre, MortarMatrix,
+                  VectorAnalysisNnodes, AnalysisVandermonde} <: AbstractDg{1, POLYDEG}
+  equations::Eqn
+
+  surface_flux_function::SurfaceFlux
+  volume_flux_function::VolumeFlux
+
+  initial_conditions::InitialConditions
+  source_terms::SourceTerms
+
+  elements::ElementContainer1D{NVARS, POLYDEG}
+  n_elements::Int
+
+  interfaces::InterfaceContainer1D{NVARS, POLYDEG}
+  n_interfaces::Int
+
+  boundaries::BoundaryContainer1D{NVARS, POLYDEG}
+  n_boundaries::Int
+  n_boundaries_per_direction::SVector{2, Int}
+
+  n_l2mortars::Int # TODO: Only needed for simulation summary output -> fix me when Taal is alive
```

Comment:
```suggestion
  n_l2mortars::Int # TODO: Taal. Only needed for simulation summary output -> fix me when Taal is alive
```
Then it will be easier for me to see that this has to be changed once this is merged into master and #187.